README.md

```diff
@@ -142,14 +142,14 @@ You specify these parameters by providing go-spacemesh with a json config file.
 3. Start go-spacemesh with the following arguments:
 
 ```bash
-./go-spacemesh --tcp-port [a_port] --config [configFileLocation] -d [nodeDataFilesPath]
+./go-spacemesh --tcp-port [a_port] --grpc-server --json-server --config [configFileLocation] -d [nodeDataFilesPath]
```

Comment:
These instructions won't work with the develop branch as they've changed for the new API. Can we leave these changes out? I'll write a separate doc explaining how to use the API